app/Models/Comment.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use App\Models\User;
+use App\Models\Post;
+
+class Comment extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [ 'user_id', 'post_id', 'content' ];
```

Comment:
Cái này sai convention mà CI ko báo lỗi =))
```suggestion
    protected $fillable = [
        'user_id',
        'post_id',
        'content',
    ];
```

Reply:
À anh nhầm, viết như kia vẫn đúng :v